src/importers.rst

```diff
@@ -4,7 +4,7 @@ Importers
 ActivityWatch can't track everything, so sometimes you might want to import data from another source.
 
 - :gh-aw:`aw-import-ical`, supports importing from ``.ical`` files or syncing with Google Calendar.
-- :gh-aw:`aw-importer-smartertime`, imports from `smartertime`_ (Android time tracker).
+- :gh-aw:`aw-importer-smartertime`, imports from `smartertime`_ (Deprecated in favour of - :gh-aw:`aw-android`).
```

Comment:
I don't know if "deprecated" is the right term. I still use it while we're working on making aw-android more accurate :)

Reply:
What would be a better way to phrase it?